spyder_notebook/widgets/client.py

```diff
@@ -204,6 +204,23 @@ def save(self):
         """Save current notebook."""
         self.notebookwidget.click("#save-notbook button")
 
+    def get_kernel_id(self):
+        """Get the kernel id and the sessions url of the client."""
+        sessions_url = self.add_token(url_path_join(self.server_url,
+                                                    'api/sessions'))
+        sessions_req = requests.get(sessions_url).content.decode()
+        sessions = json.loads(sessions_req)
+
+        if os.name == 'nt':
+            path = self.path.replace('\\', '/')
+        else:
+            path = self.path
+
+        for session in sessions:
+            if session['notebook']['path'] == path:
+                kernel_id = session['kernel']['id']
+                return (kernel_id, sessions_url)
```

Comment:
This is a method that I create initialy for a test (it was on `test_plugin.py` as a utility function). For the test where is used it, `kernel_id` and `session_id` are needed, should I rename it with something like `get_kernel_info`?